TimberWinR.UnitTests/FakeRediServer.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Net;
+using System.Net.Sockets;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using TimberWinR.Parser;
+using System.Text;
+using System.Collections.Generic;
+
+namespace TimberWinR.UnitTests
+{   
+    // Class which implements a Fake redis server for test purposes.
+    class FakeRediServer
```

Comment:
You mentioned in a face to face conversation that you found a reference fake redis server online. Is it worth including a link back to the original source?